.github/workflows/ci_python.yml

```diff
@@ -345,20 +420,23 @@ jobs:
         id: deployment
         uses: actions/deploy-pages@v2
 
-  # release:
-  #   name: Release
-  #   runs-on: ubuntu-latest
-  #   if: "startsWith(github.ref, 'refs/tags/')"
-  #   needs: [linux, windows, macos, sdist]
-  #   steps:
-  #     - uses: actions/download-artifact@v3
-  #       with:
-  #         name: wheels
-  #     - name: Publish to PyPI
-  #       uses: PyO3/maturin-action@v1
-  #       env:
-  #         MATURIN_PYPI_TOKEN: ${{ secrets.PYPI_API_TOKEN }}
-  #       with:
-  #         command: upload
-  #         args: --skip-existing *
-  #         working-directory: python
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    environment: pypi
+    needs: [build-wheel-linux, build-wheel-windows, build-wheel-macos, sdist]
+    permissions:
+      id-token: write
+    if: github.event_name == 'release' && github.event.action == 'published'
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          # unpacks default artifact into dist/
+          # if `name: artifact` is omitted, the action will create extra parent dir
+          name: artifact
+          path: dist
+
+      # - uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
Currently, I have the actual publishing disabled. I'll probably add it once I get things working.